gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTableJSONSerializer.java

```diff
@@ -41,6 +52,7 @@
   // SimpleDateFormat is not thread safe must use as a ThreadLocal
   static final ThreadLocal<DateFormat> JSON_DATE_FORMAT = ThreadLocal.withInitial(() ->
        new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.getDefault()));
+  private static final Set<PosixFilePermission> ONLY_OWNER_FILE_PERMISSIONS = PosixFilePermissions.fromString("rw-------");
```

Comment:
This will only work on UNIX. This will break on Windows. Better to use owner canRead/canWrite/etc there are ways to set this on a Path/File without using `PosixFilePermissions`

Reply:
thanks, will fix as soon as I can

Reply:
Fixed

Reply:
Hmmm I still see `PosixFilePermission*` here?